### ISSUE TITLE
'BatchNorm2d' does not match 'BatchNorm1d'

### ISSUE THREAD

**Issue by daixiangzi** [microsoft/nni#4832]
config_list = [{
        'total_sparsity': 0.5,
        'op_types': ['BatchNorm1d'],
        'max_sparsity_per_layer': 0.9
    }]
    traced_optimizer = nni.trace(torch.optim.Adam)(model_train.parameters(), lr=0.01, weight_decay=5e-4)
    pruner = SlimPruner(model_train, config_list, trainer, traced_optimizer, criterion, training_epochs=1, scale=0.0001, mode='global')

sliming not support bn1 layer?

**Comment by J-shang** [comment]
hello @daixiangzi ,you could add `BatchNorm1d` here to pass the schema check, slim pruner can support it,
https://github.com/microsoft/nni/blob/cbb63c5bec618354a25583c0861f45d4a01d9812/nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py#L545
